mobilecoind/README.md

```diff
@@ -1,10 +1,10 @@
-## The MobileCoin Daemon
+## mobilecoind
 
-`mobilecoind` is a standalone executable, which provides blockchain synchronization and wallet services.
+The MobileCoin Daemon is a standalone executable, `mobilecoind`, which provides blockchain synchronization and wallet services.
```

Comment:
"The MobileCoin Daemon, or `mobilecoind`, is a standalone executable which provides..."